src/dispatch/static/dispatch/tests/participants-tab.spec.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @vitest-environment jsdom
+ */
+
+import Vuex from "vuex"
+import { describe, expect, it, vi } from "vitest"
+import { createLocalVue, mount } from "@vue/test-utils"
+import CaseParticipantsTab from "src/case/ParticipantsTab.vue"
+
+vi.mock("vuex-map-fields", () => ({
+  getterType: vi.fn(),
+  mapFields: vi.fn(),
+}))
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+
+describe("CaseParticipantsTab", () => {
+  let store
+  let state
+
+  beforeEach(() => {
+    state = {
+      case_management: {
+        selected: {
+          participants: [
+            {
+              id: 1,
+              individual: {
+                name: "John Doe",
+                weblink: "https://example.com/john-doe",
+              },
+              participant_roles: [
+                { role: "Participant", renounced_at: null },
+                { role: "Reporter", renounced_at: "2022-01-01T00:00:00.000Z" },
+              ],
+              team: "Team A",
+              location: "Location A",
+            },
+            {
+              id: 2,
+              individual: {
+                name: "Jane Doe",
+                weblink: "https://example.com/jane-doe",
+              },
+              participant_roles: [{ role: "Assignee", renounced_at: null }],
+              team: "Team B",
+              location: "Location B",
+            },
+          ],
+        },
+      },
+    }
+
+    store = new Vuex.Store({ state })
+  })
+
+  it("renders the correct number of participants", () => {
+    const wrapper = mount(CaseParticipantsTab, { store, localVue })
+    const participants = wrapper.vm.$store.state.case_management.selected.participants
+    expect(participants).to.have.lengthOf(2)
```

Comment:
Doesn't this just check that we have an array of two in the participant store (which you set directly above)? I don't see how this actually checks to see if the dom actually rendered two participants.

Reply:
> @kevgliss @mvilanova Any opinions on where the tests for front-end should be stored? Here they are stored in the static/ dir and E2E tests are currently in the gitroot tests/ dir. Which do you prefer?

I'm fine with those locations, they make sense if asking a question "what do these tests cover"? e.g. integration is a top level but various unit or lower level tests are located closer to the code under test.

Reply:
Yeah I think the test is wrong. I'll dig into it more.

Reply:
Should be fixed now.